### ISSUE TITLE
Support for Playwright

### ISSUE THREAD

**Issue by ganeshsirsi** [pa11y/pa11y#607]
I suggest you support the pa11y framework for playwrights as well, Since many organizations switched to Playwright, we cannot depend on Puppeteer only for Accessibility, pa11y has got all good features but our framework stopping us to integrate.

**Comment by venkatsaivikram** [comment]
Hope this [comment](https://github.com/pa11y/pa11y/issues/534#issuecomment-3060831334) helps..
